semaphore/backstop.sh

```diff
@@ -79,7 +79,7 @@ else
     BRANCH=$(git rev-parse --abbrev-ref HEAD)
     FILENAME="$BRANCH.tar.gz"
     tar -czvf "$FILENAME" apps/site/test/backstop_data
-    LINK=$(curl -F "file=@$FILENAME" https://file.io/\?expires\=1d | jq -r .link)
+    LINK=$(curl -F "file=@$FILENAME" https://file.io/\?expires=1d | jq -r .link)
```

Comment:
ShellCheck pointed out we don't need to escape this as `=` is not a special character in this context.